tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-ortmodule-distributed-test-ci-pipeline.yml

```diff
@@ -0,0 +1,55 @@
+trigger: none
```

Comment:
after getting into master, we need to merge this pipeline with orttraining-distributed pipeline.

Reply:
I will create a task for this. We may need to increase number of VMs for it

Reply:
I already have a task created for this. https://msdata.visualstudio.com/Vienna/_workitems/edit/1069645